.github/workflows/ci.yaml

```diff
@@ -63,10 +63,13 @@ jobs:
       - name: build and import image
         run: |
           make images
-          docker save everoute/release | sudo k3s ctr images import -
+          docker tag everoute/release everoute/release:latest
+          docker save everoute/release:latest | sudo k3s ctr images import -
```

Comment:
why retag from everoute/release to everoute/release:latest ？

Reply:
image in controller.yaml & agent.yaml were set to everoute/release:latest. Only latest tag will re-pull during each installation.